packages/chart/src/scss/main.scss

```diff
@@ -82,6 +82,12 @@
     margin-top: 3em;
   }
 
+  .type-pie {
```

Comment:
This is impacting all svgs on the page under type-pie, ie. the data table svg plus/minus icon.